lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -482,46 +707,33 @@ bool SessionStateEnforcer::static_rule_init() {
   reqpdr1.set_precedence(32);
   reqpdr1.set_pdr_version(1);
   reqpdr1.set_pdr_state(PdrState::INSTALL);
+  reqpdr1.mutable_pdi()->set_src_interface(ACCESS);
 
-  reqpdr1.mutable_pdi()->set_ue_ip_adr("192.168.128.11");

```

Comment:
where is the ue ip address is communicated after removal of the hardcoded value?

Reply:
Looks like you are not reviewing latest version of this file, Its older version. These all removed long back.